pkg/controller/cluster/cluster_controller_test.go

```diff
@@ -319,27 +346,26 @@ func (ea expectedNodeGroupDeleteAction) validate(t *testing.T, action clientgote
 	return strings.HasPrefix(deleteAction.GetName(), ea.namePrefix)
 }
 
-// newExpectedMasterNodeGroupDeleteAction creates a new expected client
-// action for deleting a master node group.
-func newExpectedMasterNodeGroupDeleteAction(cluster *clusteroperator.Cluster) expectedNodeGroupDeleteAction {
-	return expectedNodeGroupDeleteAction{
-		namePrefix: getNamePrefixForMasterNodeGroup(cluster),
+// newExpectedMasterMachineSetDeleteAction creates a new expected client
+// action for deleting a master machine set.
+func newExpectedMasterMachineSetDeleteAction(cluster *clusteroperator.Cluster) expectedMachineSetDeleteAction {
+	return expectedMachineSetDeleteAction{
+		namePrefix: getNamePrefixForMasterMachineSet(cluster),
 	}
 }
 
-// newExpectedComputeNodeGroupDeleteAction creates a new expected client
-// action for deleting a compute node group.
-func newExpectedComputeNodeGroupDeleteAction(cluster *clusteroperator.Cluster, computeName string) expectedNodeGroupDeleteAction {
-	return expectedNodeGroupDeleteAction{
-		namePrefix: getNamePrefixForComputeNodeGroup(cluster, computeName),
+// newExpectedComputeMachineSetDeleteAction creates a new expected client
+// action for deleting a compute machine set.
+func newExpectedComputeMachineSetDeleteAction(cluster *clusteroperator.Cluster, computeName string) expectedMachineSetDeleteAction {
+	return expectedMachineSetDeleteAction{
+		namePrefix: getNamePrefixForComputeMachineSet(cluster, computeName),
 	}
 }
 
 // expectedClusterStatusUpdateAction is an expected client action to update
 // the status of a cluster.
 type expectedClusterStatusUpdateAction struct {
-	masterNodeGroups  int
-	computeNodeGroups int
+	machineSets int
```

Comment:
We can add this later, but we will want to verify `MasterMachineSetName` and `InfraMachineSetName` in this expected action as well.